L/LAMMPS/build_tarballs.jl

```diff
@@ -58,12 +59,38 @@ products = [
     ExecutableProduct("lmp", :lmp),
 ]
 
+mpi_abis = (
+    (:mpich, PackageSpec(name="MPICH_jll"), "") ,
+    (:msmpi, PackageSpec(name="MicrosoftMPI_jll"), ""),
+    (:mpitrampoline, PackageSpec(name="MPItrampoline_jll"), "2")
+)
+
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll")),
-    Dependency(PackageSpec(name="MPItrampoline_jll"), compat="2"),
-    Dependency(PackageSpec(name="MicrosoftMPI_jll"))
+    # Dependency(PackageSpec(name="MPIPlatformTag")),
 ]
 
+all_platforms = AbstractPlatform[]
+for (abi, pkg, compat) in mpi_abis
+    pkg_platforms = deepcopy(platforms)
```

Comment:
@giordano this deepcopy was necessary xD Otherwise we would mutate the platforms and they would all be the same.
Caused the issue I told you about on slack,